Code/lib/corner/corner.h

```diff
@@ -0,0 +1,28 @@
+#ifndef CORNER_H
+#define CORNER_H
+
+typedef enum{
+  LEFT,
+  RIGHT,
+  FORWARD,
+  BRAKE,
+}turn_direction;
+
+
+class CornerMerchantry
+{
+private:
+    bool wall_detected(float, float);
```

Comment:
Name parameters for readability

Reply:
in the interest of uniformity, this is now done